src/collective/exportimport/import_other.py

```diff
@@ -244,7 +244,12 @@ def import_members(self, data):
                     u"Skipping user {} without email: {}".format(username, item)
                 )
                 continue
-            pr.addMember(username, password, roles, [], item)
+            try:
+                pr.addMember(username, password, roles, [], item)
+            except ValueError:
+                logger.info(
+                    u"ValueError {} : {}".format(username, item)
+                )
```

Comment:
I think you want to add `continue` at the end of the exception handling. Otherwise adding the non-existing user to a group a few lines further on will fail.